docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_privatelink/_index.md

```diff
@@ -0,0 +1,234 @@
+
+    ```shell
+    az login
+    az ad sp create-for-rbac -n "<Unique SP Name>" --role contributor
+    ```
+
+    For example:
+
+    ```shell
+    az ad sp create-for-rbac -n "http://AzureArcServers" --role contributor
+    ```
+
+    Output should look like this:
+
+    ```json
+    {
+    "appId": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "displayName": "AzureArcServers",
+    "name": "http://AzureArcServers",
+    "password": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "tenant": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+    }
+    ```
+
+  > **Note: It is optional but highly recommended to scope the SP to a specific [Azure subscription and resource group](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest).**
+
+## Automation Flow
+
+For you to get familiar with the automation and deployment flow, below is an explanation.
+
+1. User is editing the ARM templates parameters file (1-time edit). These parameters values are being used throughout the deployment. Two separate files are provided, one per resource group.
+
+2. User creates two resource groups in two different regions:
+    * Azure Resource Group
+    * On-premises Resource Group
+
+3. User runs the ARM template for the resources in the Azure Resource Group. The ARM template will create
+
+    * Azure SQL server and a SQL database
+
+    * Azure Private Link
+
+    * Azure Private DNS zone
+
+    * Azure VPN Gateway and its public IP address
+
+    * Azure VNET
+
+4. User runs the ARM template for the resources in the On-premises resource group. The ARM template will create: 
+
+    * Azure VNET
+
+    * Azure Bastion
+
+    * Azure VPN Gateway and its public IP address
+
+    * Azure Linux Virtual Machine with a custom script extension that will run the [*install_arc_agent.sh*](https://github.com/microsoft/azure_arc/blob/main/azure_arc_servers_jumpstart/azure/linux/arm_template/scripts/install_arc_agent.sh) shell script to Arc-enable the Azure VM.
+
+        > **Note: The [*install_arc_agent.sh*](https://github.com/microsoft/azure_arc/blob/main/azure_arc_servers_jumpstart/azure/linux/arm_template/scripts/install_arc_agent.sh) shell script will enable the OS firewall and set up new rules for incoming and outgoing connections. By default all incoming and outgoing traffic will be allowed, except blocking Azure IMDS outbound traffic to the *169.254.169.254* remote address.**
+
+5. User uses Azure Bastion to connect to Linux VM which will start the *install_arc_agent.sh* script execution and will onboard the VM to Azure Arc.
+
+6. User tests private connectivity to Azure SQL server from the Azure Linux Virtual Machine.
+
+## Deployment
+
+As mentioned, this deployment will leverage ARM templates. You will deploy an ARM template per resource group that are responsible for creating all the Azure resources as well as the "on-premises" resources that will be onboarded to Azure Arc.
```

Comment:
```suggestion
As mentioned, this deployment will leverage ARM templates. You will deploy an ARM template per resource group that is responsible for creating all the Azure resources as well as the "on-premises" resources that will be onboarded to Azure Arc.
```

Can't we use a single template/param file using nested templates? makes the user experience much easier and less confusing IMO

Reply:
Will continue reviewing following that decision

Reply:
I did try but the dependencies to do the VPN connection and since things are deployed in 2 different resources groups I could not make it work.